nodejs/nodejs-guestbook/kubernetes-manifests/guestbook-backend.deployment.yaml

```diff
@@ -36,3 +36,10 @@ spec:
               key: password
         - name: MONGO_HOST
           value: "mongo-service"
+        readinessProbe:
+          httpGet:
+              port: 8080
+              path: /messages
+          failureThreshold: 3
```

Comment:
maybe reduce threshold to 1 so that app crashes right away and Kubecode isn't tricked into thinking "app has stayed up for N seconds" so it sees the crash and keeps waiting until the pod is ready. (if you make it 1, make `initialDelaySeconds: 2`)

that said, if you validated this works OK for the demo and it's successful, keep it.

Reply:
It never crashed for me with these settings - neither at docker for desktop nor at gcp

Reply:
it's because it's set to 3, and mongodb starts within 3x10 seconds.

As long as kubecode didn't call the deployment "Ready" and gave you a URL within that timeframe, it's ok.